### PR TITLE
zsxd: update repo location

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
only the homepage audit failure

```
 * The homepage URL https://www.solarus-games.org/en/games/the-legend-of-zelda-mystery-of-solarus-xd is not reachable (HTTP status code 404)
```

File: Formula/zsxd.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Zsxd < Formula
   desc "Zelda Mystery of Solarus XD"
   homepage "https://www.solarus-games.org/en/games/the-legend-of-zelda-mystery-of-solarus-xd"
```
Comment (Bo98) [inline Formula/zsxd.rb:3]:
```suggestion
  homepage "https://www.solarus-games.org/games/the-legend-of-zelda-mystery-of-solarus-xd/"
```

Comment (chenrui333) [reply Formula/zsxd.rb:3]: You are right, somehow browser did the magic redirection.

Comment (Bo98) [reply Formula/zsxd.rb]: Yeah it was a 404 with a `meta http-equiv="refresh"` tag. Bit of an odd way to do it rather than a 301.

**Review by Bo98 — APPROVED** [review]

**Pull request merged**